sign/gcsca/gcsca.go

```diff
@@ -318,36 +328,52 @@ func certPemBytes(cert *x509.Certificate) []byte {
 	return pem.EncodeToMemory(&pem.Block{Type: "CERTIFICATE", Bytes: cert.Raw})
 }
 
+type overwritten struct {
+	name string
+}
+
+type newobj struct {
+	name string
+}
+
 // upload uploads the given x.509 certificate for the named key to the GCS private bucket,
-// and updates the given manifest to reflect the created object's entry. Returns whether any writes
-// happened and any errors.
-func (ca *CertificateAuthority) upload(ctx context.Context, manifest *cpb.GCECertificateManifest, keyVersionName string, cert *x509.Certificate) (bool, error) {
+// and updates the given manifest to reflect the created object's entry. Returns whether any manifest
```

Comment:
Comment is out of date -- no longer "returns whether any manifest changes happened"